cloud

```diff
@@ -1323,6 +1323,7 @@ function cloud_functions_env() {
                           smtp_to_addrs smtp_password_secret \
                           smtp_topic smtp_subscription \
                           pgpass_secret \
+                          cache_bucket_name \
```

Comment:
This should show up in "Args:", don't it?

Reply:
okay

Reply:
We still need the argument in the function documentation, as @tales-aparecida hinted:
> This should show up in "Args:", don't it?